README.md

```diff
@@ -0,0 +1,21 @@
+# New XKit Organization
+
+New XKit is the community that has taken over the operation and maintenence of the XKit extension framework for Tumblr.
+
+This is the organization repository for collecting administrative information and records associated with the New XKit project.
+
+# Community
+
+The New XKit community is spread across various services.  Below is a list of official communication channels and accounts.
+
+- [GitHub](https://github.com/new-xkit)
+  - [new-xkit/XKit](https://github.com/new-xkit/XKit) - Fork of XKit, the extension framework for [Tumblr](https://www.tumblr.com/)
+  - [new-xkit/XCloud](https://github.com/new-xkit/XCloud) - XCloud for New XKit.  Save and sync settings across machines
+- Tumblr
+  - [new-xkit-extension](http://new-xkit-extension.tumblr.com/) - Announcements and official statements will come from this blog
+  - [new-xkit-support](http://new-xkit-support.tumblr.com/) - Need help with XKit?  Have a question or feature idea?  Send us [an ask](http://new-xkit-support.tumblr.com/ask)
+  - [new-xkit-discussion](http://new-xkit-discussion.tumblr.com/) - More relaxed blog.  Come hang out and share cool stuff with us
+- Gitter
+  - [Support chat](https://gitter.im/new-xkit/support) - Need help with XKit?  Have a question?  Join us in our support channel
+  - [Dev chat](https://gitter.im/new-xkit/XKit) - Want to help develop XKit?  Come chat with the developer community!
```

Comment:
The inconsistency here is that the other descriptions don't end with punctuation. Should we add periods, etc. to the end of the other descriptions or remove the exclamation mark here? Or something else?